implementations/micrometer-registry-wavefront/src/main/java/io/micrometer/wavefront/WavefrontConstants.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2017 Pivotal Software, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.wavefront;
+
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.Tag;
+
+/**
+ * Class containing Wavefront-specific constants and related static methods.
+ *
+ * @author Han Zhang
+ */
+public class WavefrontConstants {
```

Comment:
Please add private constructor and final modifier to the class to prevent instance creation and class inheritance

Reply:
Updated.